defaults/main.yml

```diff
@@ -23,6 +23,9 @@ aiida_postgres_db_pw: "ne9N_LDK-*JSS"
 aiida_postgres_db_user: aiida
 aiida_profile_name: generic
 
+# if true only install and configure a clean AiiDA setup, not including plugins or auxiliaries
+aiida_clean: true
```

Comment:
would it be possible to replace this with something more configurable?

E.g. 
```
aiida_components:
  - plugins
  - pseudopotentials
  - examples
```

And the checks with
```
when: plugins in aiida_components
```

Reply:
Yes, that is a good suggestion. Added.